### PR TITLE
Added credentialSchema details.

### PR DESCRIPTION
**Pull request by rghiorghisor**

### CODE REVIEW

**Comment by rghiorghisor** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by rghiorghisor** [comment]
recheck

File: docs/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -228,6 +228,10 @@ A temporary _<<C2PA assertion>>_ that is created during _<<C2PA claim>>_ generat
 
 The set of _<<_c2pa_assertion,C2PA assertions>>_ that are referenced by an *<<_identity_assertion,identity assertion>>* and thus bound to (i.e. authorized by or created by) the _<<_credential_holder,credential holder>>_ named in the *<<_identity_assertion,identity assertion>>.*
 
+==== Identity verifier
+
+An organization or system that is responsible with confirming the authenticity and validity of an individual's or entity's identity.
```
Comment (scouten-adobe) [inline docs/modules/ROOT/pages/index.adoc:233]: Wording suggestion:

> An organisation or system that is responsible _for_ confirming ...

**Comment by scouten-adobe** [comment]
Closed in favor of #126 (same code, different repo)

**Pull request closed without merging**